<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newFilm</class>
 <widget class="QMainWindow" name="newFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMDB #</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Added</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Completed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location/Platform</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IMDB_Number">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - /tt0468569/ after title in IMDB URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filmRating">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>G, PG, PG-13, R</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filmCompletion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Watched in full?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filmLocation">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Theater, Home, streaming, physcial media</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateAdded">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchFilm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilm">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Film</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="filminfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1. Input IMDB ID # for film being added</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>690</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5. Click 'Add Film' to add to database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>630</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2. Hit 'Search' button above</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>670</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4. Film out watch session details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>555</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>630</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>650</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3. Review results</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
